package.json

```diff
@@ -4,8 +4,8 @@
   "description": "",
   "main": "src/index.ts",
   "engines": {
-    "node": "10.*",
-    "npm": "6.*"
+    "node": "16.*",
```

Comment:
I think AWS is still on node 14 https://docs.aws.amazon.com/lambda/latest/dg/lambda-nodejs.html

Reply:
That is a pain! I'll have to downgrade @dvsa/eslint-config-ts from 2.3.0 to 2.2.0.